### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by artforlife**
Included npm start.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ Listen to YouTube videos, without the distracting visuals.
 Clone this repository. Run `npm install` from within the cloned directory. Amongst other npm modules, this will install `electron-prebuilt` module. To launch the app, use `./node_modules/.bin/electron .` from within the directory. 
 If your Electron is installed globally, you can also use `electron .`
 
+Alternatively, one can start the app with `npm start` after running `npm install`.
```
Comment (shakeelmohamed) [inline README.md]: This should be the only documented way of starting the app

File: README.md [hunk 2]
```diff
@@ -10,8 +10,5 @@ Listen to YouTube videos, without the distracting visuals.
 
 
 ## To Run the Desktop Version
-Clone this repository. Run `npm install` from within the cloned directory. Amongst other npm modules, this will install `electron-prebuilt` module. To launch the app, use `./node_modules/.bin/electron .` from within the directory. 
-If your Electron is installed globally, you can also use `electron .`
-
-Alternatively, one can start the app with `npm start` after running `npm install`.
+Clone this repository. Run `npm install` from within the cloned directory. Amongst other npm modules, this will install `electron-prebuilt` module. To start the app execute `npm start` after running `npm install`.
```
Comment (shakeelmohamed) [inline README.md]: `after running npm install.` at the end is a verbose, clean that up and we're good

**Comment by artforlife** [comment]
Done.

**Comment by shakeelmohamed** [comment]
I'll squash & cherry-pick this in, don't worry about the conflicts

**Comment by shakeelmohamed** [comment]
closed in favor of #7

**Pull request closed without merging**